packages/web-components/src/components/va-pagination/va-pagination.tsx

```diff
@@ -89,6 +109,68 @@ export class VaPagination {
     }
   };
 
+  private makeArray(start, end) {
```

Comment:
Would it be more useful for this to be added to the utilities file?

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/utils/utils.ts

Reply:
done!